frontend/src/components/Step4.tsx

```diff
@@ -0,0 +1,154 @@
+                />
+              </label>
+
+              <label htmlFor="licenseExpiration" className={styles.label}>
+                Date Certified/Registered/License Expires
+                <input
+                  className={styles.input}
+                  type="text"
+                  name="licenseExpiration"
+                  id="licenseExpiration"
+                  placeholder="Enter Date Certified/Registered/License Expires"
+                />
+              </label>
+            </div>
+
+            <div className={styles.formRow}>
+              <label htmlFor="certificationExam" className={styles.label}>
+                Certification Exam
+                <input
+                  className={styles.input}
+                  type="text"
+                  name="certificationExam"
+                  id="certificationExam"
+                  placeholder="Enter Certification Exam"
+                />
+              </label>
+
+              <label htmlFor="anwser" className={styles.label}>
+                Have you ever been convicted of a Felony?
+                <Dropdown options={["Yes", "No"]}></Dropdown>
+              </label>
+            </div>
+
+            <div className={styles.formRow}>
+              <label htmlFor="dateOfCertificationExam" className={styles.label}>
+                Date of Certification Exam
+                <input
+                  className={styles.input}
+                  type="text"
+                  name="dateOfCertificationExam"
+                  placeholder="mm/dd/yyyy"
+                  id="dateOfCertificationExam"
+                  pattern="\d{2}-\d{2}-\d{4}"
+                />
+              </label>
+
+              <label htmlFor="explanation" className={styles.label}>
+                If yes, please explain
+                <input
+                  className={styles.input}
+                  type="text"
+                  name="explanation"
+                  id="explanation"
+                  placeholder="Please explain here"
+                />
+              </label>
+            </div>
+
+            <hr className={styles.line} />
+
+            <h2 className={styles.sectionTitle}>
+              Rules & Regulations<span className={styles.red}>*</span>
+            </h2>
+            <p className={styles.note}>
+              Please read the{" "}
+              <a
+                className={styles.lightBlue}
+                href="https://ccidc.org/wp-content/uploads/2017/07/CCIDC-Rules-and-Regulations.pdf"
+              >
+                CCIDC RULES & REGULATIONS
+              </a>
+            </p>
+
+            <div className={styles.checkBox}>
+              <input className={styles.select} type="checkbox" id="select" required />
```

Comment:
![image](https://github.com/TritonSE/CCIDC-Application-Overhaul/assets/59126064/b9d5b2fb-1e7c-42c7-ad60-5385a6b0d5a9)
The box isn't really square when the text overflows.